librz/core/cmd/cmd_debug.c

```diff
@@ -3904,6 +3658,108 @@ RZ_IPI int rz_cmd_debug_continue_until(void *data, const char *input) {
 	return RZ_CMD_STATUS_OK;
 }
 
+// ds
+RZ_IPI RzCmdStatus rz_cmd_debug_step_handler(RzCore *core, int argc, const char **argv) {
+	const int times = argc > 1 ? (int)rz_num_math(core->num, argv[1]) : 1;
+	bool ret = rz_core_debug_step_one(core, times);
+	rz_core_dbg_follow_seek_register(core);
+	return bool2status(ret);
+}
+
+// dsb
+RZ_IPI RzCmdStatus rz_cmd_debug_step_back_handler(RzCore *core, int argc, const char **argv) {
+	CMD_CHECK_DEBUG_DEAD(core);
+	const int times = argc > 1 ? (int)rz_num_math(core->num, argv[1]) : 1;
```

Comment:
```suggestion
	const int times = (int)rz_num_math(core->num, argv[1]);
```
you can just assume `1`, the default value, will be put automatically by the system.